modules/transport-netty4/src/main/java/org/elasticsearch/http/netty4/Netty4HttpRequest.java

```diff
@@ -249,6 +189,46 @@ public Exception getInboundException() {
         return inboundException;
     }
 
+    protected io.netty.handler.codec.http.HttpRequest getNettyRequest() {
+        return request;
+    }
```

Comment:
I need in order to get to the netty `HttpHeaders` that contains the authenticated context.